src/main/java/teammates/ui/controller/InstructorFeedbackResponseCommentAddAction.java

```diff
@@ -124,7 +124,9 @@ protected ActionResult execute() throws EntityDoesNotExistException {
         data.commentId = commentId;
         data.showCommentToString = joinParticipantTypes(createdComment.showCommentTo, ",");
         data.showGiverNameToString = joinParticipantTypes(createdComment.showGiverNameTo, ",");
-        
+
+        data.setCommentIds(commentId);
```

Comment:
Current implementation initializes commentIds only while calling getComment() method, so I needed a way to initialize it before passing to .jsp